ts/util/html-utils.ts

```diff
@@ -1,5 +1,5 @@
 // adds a header cell to a table header
-export function addColumnHeader(
+export function makeColumnHeader(
```

Comment:
Please rename back to "add" for clarity since it does in fact add the element to the DOM